app/src/main/java/com/pokemonshowdown/data/ItemDex.java

```diff
@@ -90,12 +54,14 @@ public String getItemName(String name) {
         return item;
```

Comment:
I thought about it. We should be fine returning null for item name. To avoid confusion, let's rename method parameter to `String itemId`. And they should be itemId without prefix. I don't recall using prefix with at any place in the past.